dtmi/vecow/aic_100-1.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
@chrishuang19790528 -- since @WayneHou118 first created the `dtmi/vecow/*` namespace can we have them sign-off on this pr?

Reply:
yes, we are the same team at Vecow.